common/common.go

```diff
@@ -340,3 +340,17 @@ func ChangePermission(directory string) error {
 		return nil
 	})
 }
+
+// SplitStringSliceByLimit splits a slice of strings into slices by input limit and returns a slice of slice of strings ¯\_(ツ)_/¯
+func SplitStringSliceByLimit(in []string, limit int) [][]string {
```

Comment:
Please add a test for this

Reply:
Test coverage has been added for function